.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+Additional snippets: [Thread](Paste the thread link here....)
```

Comment:
Include one section for Bundle size difference.
We will create an automated report for this in future, but for now this can work.